cato/client.go

```diff
@@ -170,10 +170,16 @@ func (a *CatoAdapter) handleEvent(marker string, account_id string, api_key stri
   }
 }`, account_id, marker, eventFilterString, eventSubfilterString)
 
-		success, resp := a.send(query, account_id, api_key)
-		if !success {
+		var success bool
+		var resp map[string]interface{}
+
+		for {
+			success, resp = a.send(query, account_id, api_key)
+			if success {
+				break
+			}
 			a.conf.ClientOptions.DebugLog(fmt.Sprintf("%s", resp))
-			break
+			time.Sleep(1 * time.Minute)
```

Comment:
let's do a shorter timeout (so we don't block for a long time in the event the host is shutting down), but let's also put a cap on the number of retries  (like 3?). There is already a higher level mechanism that will retry failed Adapters after a bit and auto-disable them after a while, so we don't want to duplicate the intent.